ios/RNInCallManager/RNInCallManager.m

```diff
@@ -293,6 +293,11 @@ - (void)dealloc
 
 RCT_EXPORT_METHOD(setMicrophoneMute:(BOOL)enable)
 {
+    if (enable == true) {
+        [self audioSessionSetActive:NO options:0 callerMemo:NSStringFromSelector(_cmd)];
```

Comment:
* Doesn't `audioSessionSetActive:NO` stops all the audio activity?
* Where does the var `_cmd` comes from?